File: app/src/main/java/com/sky/chowder/ui/activity/CustomActivity.java
package com.sky.chowder.ui.activity;

import android.app.Activity;
import android.os.Bundle;
import android.support.annotation.Nullable;

import com.sky.chowder.待处理.DrawView;

/**
 * Created by 李彬 on 2017/3/10.
 */

public class CustomActivity extends Activity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new DrawView(this));
    }
}

File: app/src/main/java/com/sky/chowder/ui/activity/CustomActivity.kt
package com.sky.chowder.ui.activity

import android.app.Activity
import android.os.Bundle

import com.sky.chowder.待处理.DrawView

/**
 * Created by 李彬 on 2017/3/10.
 */

class CustomActivity : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(DrawView(this))
    }
}
